osu.Game.Rulesets.Taiko/UI/TaikoPlayfield.cs

```diff
@@ -144,6 +144,7 @@ private void load(OsuColour colours)
                     RelativeSizeAxes = Axes.Both,
                 },
                 drumRollHitContainer.CreateProxy(),
+                // new DrumTouchInputArea(this),
```

Comment:
Was this intended to be committed?

Reply:
Removed as of 317869078f470889330702b7b99b6146569b4f75